cves/CVE-2017-5648.yml

```diff
@@ -132,8 +145,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: Based on the description in the CVE
```

Comment:
This should be a file path

Reply:
I cannot find it in the file path, but in CVE description.